tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -1144,7 +1144,11 @@ def test_fluent_autolog_with_tf_keras_preserves_v2_model_reference():
     mlflow.autolog()
 
     import tensorflow.keras
-    from keras.api._v2.keras import Model as ModelV2
+
+    if Version(tf.__version__) <= Version("2.15.0"):
+        from keras.api._v2.keras import Model as ModelV2
+    else:
+        from keras.src.models.model import Model as ModelV2
```

Comment:
Can we avoid the direct imports? importing the code path is error-prone, could `from keras import Model` work?